schemas/da00_dataarray.fbs

```diff
@@ -0,0 +1,26 @@
+
+// A flatbuffer schema for holding histogram data or EPICS area detector updates
+// A merger of hs00/hs01 and ADAr aimed at compatibility with scipp DataArrays
+
+file_identifier "da00";
+
+enum da00_dtype: byte {none, int8, uint8, int16, uint16, int32, uint32, int64, uint64, float32, float64, c_string}
+
+table da00_Variable {
+    name: string (required);   // What this data represents: e.g., 'data', 'errors', '{axis_name}'
+    unit: string;              // (optional) unit name for the represented data [expected for datasets]
+    label: string;             // (optional) label describing the data
+    source: string;            // (optional) EPICS PV name, DRV_INFO string, etc. [expected for attributes]
+    data_type: da00_dtype;     // data type for the stored data in the array
+    axes: [string] (required); // The ordered names of the axes of this data
+    shape: [long] (required);  // Shape of the multi-dimensional array
+    data: [ubyte] (required);  // C-ordered flat array interpreted as unsigned 8-bit integers
+}
+
+table da00_DataArray {
+    source_name: string (required);   // Source name of array
+    timestamp: ulong;                 // Timestamp in nanoseconds since UNIX epoch
```

Comment:
@danesss made the comment https://github.com/ess-dmsc/streaming-data-types/pull/92#discussion_r1466074499
>Consider using long type, as per recommendations in this repo
`Do not use unsigned integers unless required for your application.`
https://github.com/ess-dmsc/streaming-data-types/#schema-coding-standard

The use of `ulong` here came from [`ADAr_area_detector_rray.fbs`](https://github.com/ess-dmsc/streaming-data-types/blob/58793c3dfa060f60b4a933bc085f831744e43f17/schemas/ADAr_area_detector_array.fbs) and was not amended due, in part, to my confusion over flatbuffer data types -- namely that both `long` and `ulong` are 64-bit integers and _not_ 32-bit.

Also following the [ADAr serializer](https://github.com/ess-dmsc/python-streaming-data-types/blob/0d98d5e442c6031d7fd1e3af929e9e4389ef7a57/streaming_data_types/area_detector_ADAr.py#L119), the `timestamp` field is the number of nanoseconds since the Unix epoch.
Switching from `ulong` to `long` limits the lifetime of the `da00_DataArray` table from 584 years to _only_ 292 years. I suspect we can live with needing to revisit this timestamp issue sometime before 2262 so I will investigate implementing this change.